cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -470,7 +214,111 @@ def close(self):
-# ------------------------------------------------------------------ #
+class CephUploadTask(UploadTask):
+    def __init__(self, imageUuid, dstPath, tmpPath, ioctx):
+        super(CephUploadTask, self).__init__(imageUuid, dstPath)
+        self.tmpPath = tmpPath
+        self.ioctx = ioctx
+        self.image_format = "raw"
+
+    def complete_upload(self):
+        try:
+            file_format = linux.get_img_fmt('rbd:' + self.tmpPath)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)